hs_core/migrations/0072_remove_duplicate_res_files.py

```diff
@@ -1,12 +1,14 @@
 # Generated by Django 3.2.19 on 2023-07-27 18:50
 
-from django.db import migrations, models
-import django_irods.storage
-import hs_core.models
-from django.core.management import call_command
+import logging
+from django.db import migrations
+from hs_core.management.utils import fix_resourcefile_duplicates
+
 
 def migrate_resourcefiles(apps, schema_editor):
-    call_command('fix_resourcefile_duplicates')
+    logger = logging.getLogger(__name__)
+    fix_resourcefile_duplicates(dry_run=False, logger=logger)
```

Comment:
In the management command we should be accessing the current models (should not be using apps.get_model. I might have made this mistake.